src/scss/initial/_links.scss

```diff
@@ -68,3 +68,19 @@ a {
     text-decoration: none;
   }
 }
+
+// For links styled as buttons, underlines
+// are removed and colors match styles applied
+// buttons
+[data-btn~='link-button'] {
+  --link: var(--bg);
+  --link-focus: var(--bg);
+  --underline-color: transparent;
+  --underline-thickness: 0;
+
+  &:hover,
+  &:focus {
+    --underline-color: transparent;
+    --underline-thickness: 0px;
```

Comment:
```suggestion
    --underline-thickness: 0;
```

Reply:
Whoops. Missed that. Fixed in https://github.com/oddbird/oddleventy/pull/808/commits/a91c1fde653f587cca898006a504a8185a2da560